docker-compose-CeleryExecutor.yml

```diff
@@ -16,7 +16,7 @@ services:
         #     - ./pgdata:/var/lib/postgresql/data/pgdata
 
     webserver:
-        image: puckel/docker-airflow:1.10.9
+        image: puckel/docker-airflow:1.10.11
```

Comment:
Does it make sense to change it here even though the image does not exist in dockerhub?

Reply:
Good point, it doesn't make sense. I can change it back to puckel/docker-airflow:1.10.9. Or probably better would be to point it to quay.io/opendatahub/docker-airflow?

Reply:
I think pointing to quay.io might be good. We can go with a specific version if we want, just need to create a new tag in the registry.

Reply:
If there is a 1.10.11 tag in quay.io I could also use that tag in the airflow-on-k8s-operator.

I am also thinking to switch to using the Red Hat's database images in docker-compose-CeleryExecutor.yml and docker-compose-LocalExecutor.yml.

So, the overall strategy would be to use the same images in both docker-airflow and airflow-on-k8s-operator. I think that would make sense.

Reply:
Sure, sounds good...though I am not sure where this tag appears in airflow-on-k8s-operator

Reply:
I meant in the airflow-on-k8s-operator/api/v1alpha1/airflowcluster_types.go to change:

```
  defaultFlowerImage      = "quay.io/opendatahub/docker-airflow"
  defaultFlowerVersion    = "openshift"
  defaultSchedulerImage   = "quay.io/opendatahub/docker-airflow"
  defaultSchedulerVersion = "openshift"
  defaultUIImage          = "quay.io/opendatahub/docker-airflow"
  defaultUIVersion        = "openshift"
  defaultWorkerImage      = "quay.io/opendatahub/docker-airflow"
  defaultWorkerVersion    = "openshift"
```
to

```
  defaultFlowerImage      = "quay.io/opendatahub/docker-airflow"
  defaultFlowerVersion    = "1.10.11"
  defaultSchedulerImage   = "quay.io/opendatahub/docker-airflow"
  defaultSchedulerVersion = "1.10.11"
  defaultUIImage          = "quay.io/opendatahub/docker-airflow"
  defaultUIVersion        = "1.10.11"
  defaultWorkerImage      = "quay.io/opendatahub/docker-airflow"
  defaultWorkerVersion    = "1.10.11"
```
Plus also change this commit to use "1.10.11" instead of the "openshift" tag:
https://github.com/noseka1/airflow-on-k8s-operator/commit/3e7cdbd09a42d9c0e06908e972351035d9d816a0

Reply:
Ah, ok, yeah, sounds good

Reply:
I updated the images in both the docker-airflow and the airflow-on-k8s-operator repos.